components/error-summary/src/fixtures.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { text } from '@storybook/addon-knobs/react';
+import InputField from '@govuk-react/input-field';
+import TextArea from '@govuk-react/text-area';
+
+import ErrorSummary from '.';
+
+export const heading = 'Message to alert the user to a problem goes here';
+export const description = 'Optional description of the errors and how to correct them';
+export const errors = [
+  {
+    targetName: 'national-insurance-number',
+    text: 'National Insurance number error',
+  },
+  {
+    targetName: 'description',
+    text: 'Description of what you saw error',
+  },
+];
+
+export default () => (
```

Comment:
Not sure if it is worth `default` being the component by itself, so it can be tested and rendered in isolation without anything else? Altho, I like the tests, and I can't see any issue with this.

Reply:
Guess it might just my semantic radar getting mixed up, as it is exporting `default`, not that, the thing being exported is the components actual default... if you know what I mean? o_O 

Probably fine to leave as is